command/agent/command.go

```diff
@@ -983,9 +984,8 @@ func (c *Command) handleRetryJoin(config *Config) error {
 // also https://www.man7.org/linux/man-pages/man3/sd_notify.3.html
 const (
 	sdReady     = "READY=1"
-	sdReloading = "RELOADING=1"
+	sdReloading = "RELOADING=1\nMONOTONIC_USEC=%d"
```

Comment:
The python-equivalent from the systemd docs at [sd_notify](https://www.freedesktop.org/software/systemd/man/latest/sd_notify.html) is:

```python
def notify_reloading():
    microsecs = time.clock_gettime_ns(time.CLOCK_MONOTONIC) // 1000
    notify(f"RELOADING=1\nMONOTONIC_USEC={microsecs}".encode())
```